code/script.js

```diff
@@ -1,53 +1,273 @@
+        handleSecondChoice.preventDefault()
+        subChoice = "rye bread"
+        showMessage("I would like a rye bread, please!", "user")
+        setTimeout(() => handleThirdChoice(subChoice), 1000)
+      })
+  } else if (choice === "pastry") {
+    form.innerHTML = `<div class="sub-categories">
+    <button class="send-btn" id="semla" type="submit">Semla</button>
+    <button class="send-btn" id="cinnamon" type="submit">Cinnamon Bun</button>
+    <button class="send-btn" id="brownie" type="submit">Brownie</button></div></form>`
+
+    document
+      .getElementById("semla")
+      .addEventListener("click", (handleSecondChoice) => {
+        handleSecondChoice.preventDefault()
+        subChoice = "semla"
+        showMessage("I would like a semla, please!", "user")
+        setTimeout(() => handleThirdChoice(subChoice), 1000)
+      })
+    document
+      .getElementById("cinnamon")
+      .addEventListener("click", (handleSecondChoice) => {
+        handleSecondChoice.preventDefault()
+        subChoice = "cinnamon bun"
+        showMessage("I would like cinnamon bun, please!", "user")
+        setTimeout(() => handleThirdChoice(subChoice), 1000)
+      })
+    document
+      .getElementById("brownie")
+      .addEventListener("click", (handleSecondChoice) => {
+        handleSecondChoice.preventDefault()
+        subChoice = "brownie"
+        showMessage("I would like a brownie, please!", "user")
+        setTimeout(() => handleThirdChoice(subChoice), 1000)
+      })
+  }
+}
+
+//Function handling the amount input from user
+const handleThirdChoice = (subChoice) => {
+  showMessage(`How many ${subChoice}s would you like?`, "bot")
+  form.innerHTML = `<input id="amount-input" type="text" />
+  <button class="send-btn" id="amount-btn" type="submit">Send</button>`
+  const amountInput = document.getElementById("amount-input")
+  document.getElementById("amount-btn").addEventListener("click", (event) => {
+    event.preventDefault()
+    const amount = amountInput.value
+    showMessage(`${amount} ${subChoice}s, please!`, `user`)
+    amountInput.value = ""
+    setTimeout(() => handleFinalConfirmation(amount), 1000)
+  })
+}
+
+//Function giving the final response from bot depending on amount input from user
+const handleFinalConfirmation = (amount) => {
+  if (amount >= 1 && amount <= 10) {
+    showMessage(
+      `Thank you for your order! It will be ready in 10 minutes! ❤️`,
+      "bot"
+    )
+    setTimeout(shoot, 100)
+    form.remove()
+  } else if (amount > 10) {
+    showMessage(
+      `That's a lot! We take larger orders by phone, please call 000 123 456 ❤️`,
+      "bot"
+    )
+    form.remove()
+  } else if (amount === "0") {
+    showMessage(`Are you sure you chose the right amount?`, "bot")
+    setTimeout(() => handleThirdChoice(subChoice), 1000)
+  } else {
+    showMessage(`I didn't catch that, please try again! ❤️`, "bot")
+    setTimeout(() => handleThirdChoice(subChoice), 1000)
+  }
+}
```

Comment:
Nice logic!
How about changing the input type to number? That way - at least the user won't be able to write letters 😊